src/utils/urlBuilder.js

```diff
@@ -0,0 +1,24 @@
+import Url from 'url-parse';
```

Comment:
Not sure it is a good idea to use url-parse, it add 3 extra dependencies. createUrl function could be written elegantly and concisely without it.